custom_registration/models.py

```diff
@@ -30,7 +30,8 @@ def create_user(email, first_name, last_name, password, mailing_list_signup):
     user.save()
 
     profile = Profile(user=user)
-    profile.mailing_list_signup = mailing_list_signup
```

Comment:
@s-kennedy we can't remove the mailing_llist_signup yet, since that will mean we loose the info of who signed up to the newsletter and we want to use this to set a default of the communications_opt_in and also to know who to remove from the newsletter, since the general list will also receive newsletters.

Reply:
I'll fix the issue in a branch to remove the migration and add the field and then merge from there.